__tests__/integration/helpers/testRunner.js

```diff
@@ -46,7 +46,8 @@ const setUpTestRunner = async ({ oneAppLocalPortToUse } = {}) => {
           'one-app': {
             ports: [
               `${oneAppLocalPortToUse}:8443`,
-            ],
+              oneAppMetricsLocalPortToUse ? `${oneAppMetricsLocalPortToUse}:3005` : undefined,
```

Comment:
@JamesSingleton the 3005 port has to be exposed because it is mapped here like from ramdomPort to 3005